### ISSUE TITLE
reproducibleResults

### ISSUE THREAD

**Issue by davide1993** [autonomio/talos#177]
I use these instructions reported in keras documentation to get reproducible results:
https://keras.io/getting-started/faq/#how-can-i-obtain-reproducible-results-using-keras-during-development

As you can see multithread have to be disable but this slow down the training too much. My goal is to run a long test with talos and save different models: the best model for each metric. I think with Deploy class reproducibility is not a problem because I can use Deploy(scan object, metric) in order to save my best model for a given metric. So what I did is:
`   ta.Deploy(scan,exp_acc_filename,metric= val_acc)
    ta.Deploy(scan,exp_prec_filename,metric=val_precision_acc )
    ta.Deploy(scan,exp_rec_filename,metric= val_recall_acc)
    ta.Deploy(scan,exp_loss_filename,metric= val_loss, asc=True) 
    ta.Deploy(scan,exp_f1_filename,metric=val_fmeasure_acc )`

then I restore the model from the zip archive with 
`r=Restore('zipfile')`
r variable contains the weights, layers... so  I think there's no need to use single thread in my project because the zip file contains everything I need to reproduce the same results, am I wrong?
thank you

**Comment by mikkokotila** [comment]
That's right.

**Comment by mikkokotila** [comment]
Closing here.
